content-services/latest/config/mtls.md

```diff
@@ -0,0 +1,283 @@
+---
+title: mTLS
+---
+
+This section describes a recommended approach for generating secure keys and setting up certificates. It is not required that you use this approach if you have an alternative solution that you already use.
+
+If you're installing Content Services using the distribution zip, you need to generate certificates for the repository and Solr. By default, the distribution zips are configured to use SSL, so you'll need to generate these certificates to get your system to run successfully.
+
+You can create the keystores, truststores and certificates required to configure SSL/mutual TLS authentication between different services in Content Services, such as the repository and Solr.
+
+The diagram shown is an overview of mTLS structure:
+
+![mTLS diagram]({% link content-services/images/mtls-diagram.png %})

```

Comment:
I've tried to open this in Intellij, but it didn't seem to load the graph properly. I don't know if that means it doesn't work, or if Intellij just messes with it, but please check if it's alright :).